src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/db/ProblemRepository.java

```diff
@@ -19,7 +21,7 @@
         + "         WHEN 'Warning'  THEN '2' "
         + "         ELSE '2' "
```

Comment:
I think we shouldn't have null severity or maybe I don't know what case that fits in.

Reply:
can you elaborate? 
Rules can now produce only 3 severites, but as far I'm aware of we dont cast it to specific type in events repo. Probably it is an issue

Reply:
events and rules are loosely coupled, so do not assume any internal implementation details from rules.